docs/navigation.md

```diff
@@ -0,0 +1,114 @@
+# Navigation panel
+
+This guide will walk you through the functions of each button in navigation panel. 
```

Comment:
```suggestion
This document will walk you through the functions of each button in the navigation panel. 
```

Reply:
I wouldn't call it a guide because it won't be going under the `Guides` content bucket, and would prefer we avoid any confusion 😄